i18n-helpers/src/bin/mdbook-xgettext.rs

```diff
@@ -226,4 +422,168 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn test_create_catalog_nested_directories() -> anyhow::Result<()> {
+        let (ctx, _tmp) = create_render_context(&[
```

Comment:
If you don't need `_tmp`, then just use `_` as a wildcard pattern to throw away the binding.